src/app/landing/Testimonial.css

```diff
@@ -11,9 +11,9 @@
 }
 
 .Testimonial-image {
-  border-radius: 100%;
+  /* border-radius: 100%;
```

Comment:
I assume this is no longer needed because the images themselves were edited to match this style?
It's generally preferable to leave stylistic cropping up to CSS, as opposed to "hard-coding" the image a certain way. That way, any stylistic changes don't require re-generating all the pictures (assuming the originals are even still around).

Or did the designs actually change, s.t. we no longer want these to be round?

Reply:
That's correct, the testimonial images provided to me are already round, so this CSS was causing the halo effect to double. I'll reach out for compressed but unedited images.